packages/tokens/package.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "@vapor/tokens",
+    "private": true,
```

Comment:
IMO it should not be private, this way we'll be able to install it in the admin. 

Though, it means you'll have to change the name to `@coveord/tokens` or create a new organization in NPM or something similar

Reply:
renamed to `coveord/design-tokens` and removed `private: true`